### PR TITLE
feat: polish last bits after OdigosConfig CR removal

### PR DESCRIPTION
**Pull request by rauno56**
Check the commits for list of changes. I added `odigos.io/config` and `odigos.io/system-object` labels to all CRDs, not just the tested.

### CODE REVIEW

File: cli/pkg/autodetect/kind.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ var CurrentKubernetesVersion KubernetesVersion
 const (
 	KindUnknown   Kind = "Unknown"
 	KindMinikube  Kind = "Minikube"
-	KindKind      Kind = "KinD"
```
Comment (blumamir) [inline cli/pkg/autodetect/kind.go:25]: I think this was written like so intentionally - "**K**ubernetes in **D**ocker".
@edeNFed

File: api/actions/v1alpha1/addclusterinfo_types.go [hunk 2]
```diff
@@ -56,6 +56,8 @@ type AddClusterInfoStatus struct {
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
 //+kubebuilder:resource:path=addclusterinfos,scope=Namespaced,shortName=aci
+//+kubebuilder:metadata:labels=metadata.labels.odigos.io/config=true
+//+kubebuilder:metadata:labels=metadata.labels.odigos.io/system-object=1
```
Comment (blumamir) [inline api/actions/v1alpha1/addclusterinfo_types.go:60]: I think they are opposite, should it be?
```suggestion
//+kubebuilder:metadata:labels=metadata.labels.odigos.io/config=1
//+kubebuilder:metadata:labels=metadata.labels.odigos.io/system-object=true
```

File: cli/pkg/autodetect/kind.go (hunk 1 above)
Comment (rauno56) [reply cli/pkg/autodetect/kind.go:25]: Nowhere on their site, they use that form.

Comment (edeNFed) [reply cli/pkg/autodetect/kind.go:25]: I am ok with both

**Review by blumamir — APPROVED** [review]

**Pull request merged**